core/src/main/java/org/apache/accumulo/core/Constants.java

```diff
@@ -126,7 +126,8 @@
   public static final String EXPORT_INFO_FILE = "accumulo_export_info.txt";
 
   // Variables that will be substituted with environment vars in PropertyType.PATH values
-  public static final Collection<String> PATH_PROPERTY_ENV_VARS = Collections.unmodifiableCollection(Arrays.asList("ACCUMULO_HOME", "ACCUMULO_CONF_DIR"));
+  public static final Collection<String> PATH_PROPERTY_ENV_VARS = Collections.unmodifiableCollection(Arrays.asList("ACCUMULO_BIN_DIR", "ACCUMULO_LIB_DIR",
```

Comment:
I removed the extra interpolations and only ACCUMULO_HOME & ACCUMULO_CONF_DIR are interpolated